addon/components/listbox/-button.hbs

```diff
@@ -0,0 +1,18 @@
+{{#let (element (or @as 'button')) as |Tag|}}
+  <Tag
+    role={{if @role @role 'button'}}
```

Comment:
It doesn't seem like the upstream `Listbox.Button` component has an `@role` argument; is it necessary for us to support it?